lightning-net-tokio/Cargo.toml

```diff
@@ -11,9 +11,7 @@ For Rust-Lightning clients which wish to make direct connections to Lightning P2
 
 [dependencies]
 bitcoin = "0.21"
-bitcoin_hashes = "0.7"
```

Comment:
Does this belong in the previous commit?

Reply:
Moved